config/example_config.toml

```diff
@@ -37,6 +37,11 @@
   # Default is true.
  ## enable_relay = false
 
+  # `enable_hole_punching` indicates whether Hole Punch service  should be enabled or not.
+  # Hole Punch service is an approach to NAT traversal.
+  # Default is true.
+ ## enable_hole_punching = false
```

Comment:
I believe that when a user enables Relay, it can include HolePunching as well. This prevents conflicts between configs.
Consider a scenario where a user disables Relay but enables HolePunching.

Reply:
So, you mean we should merge the Relay config and hole punching config?